packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -570,10 +573,9 @@ const VirtualListBaseFactory = (type) => {
 
 		// Native only
 		scrollToPosition (x, y, rtl = this.props.rtl) {
+			const scrollX = (rtl && !this.isPrimaryDirectionVertical) ? this.scrollBounds.maxLeft - x : x;
 			if (this.containerRef.current) {
-				this.containerRef.current.scrollTo(
-					(rtl && !this.isPrimaryDirectionVertical) ? this.scrollBounds.maxLeft - x : x, y
-				);
+				this.containerRef.current.scrollTo(this.platformInfo.ios || this.platformInfo.safari ? -x : scrollX, y);
```

Comment:
Shouldn't this only apply -x for `rtl`?  Perhaps it would be cleaner to refactor the `scrollX` value to incorporate this so the `scrollTo()` call is simpler?

Reply:
@Baekwoo-s is working on it with fixing bug here.

Reply:
Done.